tests/IntegrationTests/WorkflowExecutor.IntegrationTests/StepDefinitions/PayloadCollectionStepDefinitions.cs

```diff
@@ -0,0 +1,59 @@
+// SPDX-FileCopyrightText: � 2021-2022 MONAI Consortium
+// SPDX-License-Identifier: Apache License 2.0
+
+using BoDi;
+using Monai.Deploy.Messaging.Events;
+using Monai.Deploy.Messaging.Messages;
+using Monai.Deploy.WorkflowManager.IntegrationTests.Models;
+using Monai.Deploy.WorkflowManager.IntegrationTests.Support;
+using TechTalk.SpecFlow.Infrastructure;
+
+namespace Monai.Deploy.WorkflowManager.IntegrationTests.StepDefinitions
+{
+    [Binding]
+    public class PayloadCollectionStepDefinitions
+    {
+        private RabbitPublisher WorkflowPublisher { get; set; }
+        private RabbitConsumer TaskDispatchConsumer { get; set; }
+        private MongoClientUtil MongoClient { get; set; }
+        private Assertions Assertions { get; set; }
+        private DataHelper DataHelper { get; set; }
+        private readonly ISpecFlowOutputHelper _outputHelper;
+
+        public PayloadCollectionStepDefinitions(ObjectContainer objectContainer, ISpecFlowOutputHelper outputHelper)
+        {
+            WorkflowPublisher = objectContainer.Resolve<RabbitPublisher>("WorkflowPublisher");

```

Comment:
Doesnt look like everyhting is used in the constructor, could remove everything but the data helper and assertions